test/unit/error.spec.ts

```diff
@@ -343,3 +344,25 @@ test("still works without debug module", () => {
             stack: undefined,
         });
 });
+
+util.testEachVersion(
+    "error stacktrace omits constructor and __TS_New",
+    () => util.testFunction`
+        const e = new Error();
+        return e.stack;
+    `,
+    {
+        ...util.expectEachVersionExceptJit(builder => {
+            builder.expectToHaveNoDiagnostics();
+            const luaResult = builder.getLuaExecutionResult();
+            expect(luaResult.split('\n').length).toBe(4);
```

Comment:
ugh, linter seems confused about our helper function here. I don't like it but I guess you can add `// eslint-disable-next-line jest/no-standalone-expect` here